src/components/App.js

```diff
@@ -1,19 +1,56 @@
-import React from 'react';
-
-import '../styles/components/app.scss';
+import React from "react";
+import Login from "./Login";
+import StudentHub from "./StudentHub";
+import ProfessionalHub from "./ProfessionalHub";
+import "../styles/components/app.scss";
 
 class App extends React.Component {
-
-  constructor(){
+  constructor() {
     super();
+    this.state = {
+      studentActive: false,
+      professionalActive: false,
+      user: "",
+      requests: [],
+      reviews: []
+    };
+    this.receiveSubmit = this.receiveSubmit.bind(this);
+  }
+  //Getting requests, setting them in state. Getting reviews, setting them in state.
+  componentDidMount() {
+    if (!localStorage.getItem("requests")) {
```

Comment:
It seems like too much work reading a value from localStorage, setting in localStorage and then reading it again. Why not read it and if it is not set then just a an empty array locally, instead of writing it to localStorage and reading it back.